src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -45,7 +45,7 @@ priority = "primary"
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 ansys-optislang-core = "0.4"
-ansys-saf-glow = {version = "0.4.0", source = "solutions-private-pypi"}
+ansys-saf-glow = {version = "0.4.1", source = "solutions-private-pypi"}
```

Comment:
@iazehaf I guess we need to update the .lock file as well ?

Reply:
Done